app/src/androidTest/assets/basicBlindTest.txt

```diff
@@ -0,0 +1,129 @@
+Rasputin:Boney M.
+September:Earth, Wind & Fire
+Sarà perché ti amo:Ricchi e Poveri
+Halo:Beyoncé
+L'aventurier:Indochine
+J'Irai Ou Tu Iras:Céline Dion;Jean-Jacques Goldman
+Africa:Toto
+I'm So Excited:The Pointer Sisters
+Besoin de rien, envie de toi:Peter & Sloane
+Cheap Thrills:Sia, Sean Paul
+Wati by Night:Sexion D'Assaut
+Beggin':Måneskin
+Another Love:Tom Odell
+Pumped Up Kicks:Foster the People
+Uptown Funk:Mark Ronson, Bruno Mars
+Wake Me Up Before You Go-Go:Wham!
+Je te donne:Jean-Jacques Goldman, Michael Jones
+Sous le vent:Céline Dion, Garou
+Billie Jean:Michael Jackson
+Sweet Dreams:Eurythmics, Annie Lennox, Dave Stewart
+Highway to Hell:AC/DC
+Come and Get Your Love :Redbone
+Dernière danse:Kyo
+Là-bas:Jean-Jacques Goldman, Sirima
+Bohemian Rhapsody:Queen
+CAN'T STOP THE FEELING!:Justin Timberlake
+Old Town Road:Lil Nas X, Billy Ray Cyrus
+Ma direction:Sexion D'Assaut
+Footloose:Kenny Loggins
+I'm Outta Love:Anastacia
+It's Raining Men:The Weather Girls
+Knockin' On Heaven's Door:Bob Dylan
+Get Lucky:Daft Punk, Pharrell Williams, Nile Rodgers
+Waka Waka:Shakira, Freshlyground
+Eye of the Tiger:Survivor
+Je t'aimais, je t'aime, je t'aimerai:Francis Cabrel
+Seven Nation Army:The White Stripes
+Ma Benz:Suprême NTM, Lord Kossity
+Hey Ya!:Outkast
+Cry Me A River:Justin Timberlake
+Shut Up and Let Me Go:The Ting Tings
+Hey, Soul Sister:Train
+Girls Just Want to Have Fun:Cyndi Lauper
+Fallin':Alicia Keys
+Thank You:Dido
+Wonderwall:Oasis
+All Of Me:John Legend
+Sexual Healing:Marvin Gaye
+Let Her Go:Passenger
+Still Loving You:Scorpions
+Should I Stay or Should I Go:The Clash
+Heavy Cross:Gossip
+Hallelujah:Jeff Buckley
+Ma philosophie:Amel Bent
```

Comment:
Actually I have only checked with few of them, but it looks like most of them work. It would be cumbersome to check them all but I agree that they should ideally all have lyrics.